### PR TITLE
Add an invert argument to find_tests_scripts

### PR DESCRIPTION
**Pull request by jimhester**
This allows selecting only tests which do _not_ match a regular
expression.

### CODE REVIEW

File: R/test-files.r [hunk 1]
```diff
@@ -62,17 +62,24 @@ source_test_helpers <- function(path, env = globalenv()) {
 #' Find the test files.
 #' @param path path to tests
 #' @param filter cf \code{\link{test_dir}}
+#' @param invert If \sQuote{TRUE} return files which do _not_ match.
```
Comment (hadley) [inline R/test-files.r]: `\emph{}`?

**Comment by hadley** [comment]
Can you add a bullet to NEWS please?

**Comment by hadley** [comment]
If you can get this in soon, it's likely to make it into the next version

**Comment by jimhester** [comment]
Done https://github.com/hadley/testthat/commit/c1fcdd01ce2ed9544f07b855a5aa181f68a48131, thanks for the reminder!

**Pull request merged**